catalog/capabilities/database/info.json

```diff
@@ -30,22 +24,10 @@
       "shared": true,
       "type": "enum",
       "values": [
-        {
-          "id": "nodejs",
-          "name": "Node.js"
-        },
-        {
-          "id": "springboot",
-          "name": "Spring Boot"
-        },
-        {
-          "id": "thorntail",
-          "name": "Thorntail"
-        },
-        {
-          "id": "vertx",
-          "name": "Vert.x"
-        }
+        "nodejs",
+        "springboot",
+        "thorntail",
+        "vertx"
       ]
     },
     {
```

Comment:
What does happen if one capability has one maven config and the other another?

Reply:
That's undefined, it's the frontend's responsibility (for now) to make sure that you pass the same value in all of them. We should adjust the API so you can pass them at the top level just as the "runtime" is passed right now. I'll open a separate issue for that.

Reply:
https://github.com/fabric8-launcher/launcher-creator-backend/issues/44